### ISSUE TITLE
:listen - not displaying an ID

### ISSUE THREAD

**Issue by tohagan** [remy/jsconsole#148]
From your docs I think I'm supposed to type `:listen` at command prompt on jsconsole.com to get a new ID and script code to insert. However, It just clears the line I typed and displays nothing.  I'm using Chrome 80.  Same in Firefox.

**Comment by rasicoc** [comment]
I am also getting this issue. I am pretty sure I am following the directions clearly. But :listen just doesn't return anything.

**Comment by Bharath722** [comment]
I am also getting this issue. Is there any update on this?

**Comment by alecmolloy** [comment]
same here

**Comment by remy** [comment]
Nope. It broke. Like…about 2+ years ago. Need to bring back up on a server that has free websocket support. If anyone has any suggestions…? Otherwise I should really get around to removing the line that actually relies…!

**Comment by samyH** [comment]
D'oh! any other suggestions for debugging iOS devices on windows?

**Comment by antonostrovsky** [comment]
same here

**Comment by tohagan** [comment]
@remy You could probably port this to Firebase and then anyone could just download and deploy their own instance. Replace logged events with a a Firestore collection or RTDB JSON. Much of your Express code would port to Firebase functions.  Large free tier so free for most devs to run their own instance. No sockets required.

**Comment by remy** [comment]
@tohagan I've never used firebase - so … probably not! All the source code is here though, there's nothing stopping anyone else from porting to firebase (if that's a useful option).

**Comment by Bharath722** [comment]
People searching to debug remotely can use [this link](https://console.re/). It allows you to debug remotely which can be helpful to debug iOS devices without mac and its free

**Comment by emyrTomos** [comment]
You need to set 'Access-Control-Allow-Origin' for https://jsconsole.glitch.me/remote/ - this is throwing a Cross-Origin Request Blocked error

**Comment by bendavis78** [comment]
Is there a way to get this working on my own server? I'm trying to find a solution to [this](https://stackoverflow.com/questions/64432592/is-there-an-alternative-to-chrome-developer-tools-for-chromebooks-that-have-it-d) issue, and this would be a lifesaver for us!

I tried cloning the repo and running "npm start", but `:listen` does not appear to work.

_**[edit]**_  I was able to make it a little further by running `node lib/app.js`. Now running `:listen` generates the script tag w/ the id token. Unfortunately that script tag references `remote.js`, which doesn't exist anywhere.  Any idea where `remote.js` is found?

_**[edit 2]**_ I was finally able to get it to work by reverting to 6ef63d7.  However, things didn't work quite like I'd expected. My hope was that I would get access to the entire context of the connected page. For example, I wanted to be able to use things like `document.querySelector`, however it doesn't seem to evaluate javascript on the connected page, only in the repl itself. Is that the expected behavior?
